package.json

```diff
@@ -22,10 +38,12 @@
     "ignore": "^5.2.4"
   },
   "devDependencies": {
+    "@arethetypeswrong/cli": "^0.16.4",
     "@babel/core": "^7.22.9",
     "@babel/eslint-parser": "^7.22.9",
     "@eslint-community/eslint-plugin-mysticatea": "^15.5.1",
-    "@types/node": "^14.18.54",
+    "@types/eslint": "^9.6.1",
+    "@types/node": "^22.8.1",
```

Comment:
I dont think we should change this until we update the `engines.node` property to drop node 12.

Reply:
```suggestion
    "@types/node": "^14.18.54",
```